handlers/settings.go

```diff
@@ -76,6 +76,27 @@ func saveSettingsHandler(w http.ResponseWriter, r *http.Request) {
 	ExecuteResponse(w, r, "settings.gohtml", core.CoreApp, "settings")
 }
 
+func saveSettingsNotificationsHandler(w http.ResponseWriter, r *http.Request) {
+	var err error
+	form := parseForm(r)
+	app := core.CoreApp
+
+	value := form.Get("update_notify")
+
+	if value == "true" {
+		app.UpdateNotify = types.NewNullBool(true)
+	} else if value == "false" {
+		app.UpdateNotify = types.NewNullBool(false)
+	} // discard if empty or other value
+
+	core.CoreApp, err = core.UpdateCore(app)
+	if err != nil {
+		log.Errorln(fmt.Sprintf("issue updating Core: %v", err.Error()))
```

Comment:
Since we are using [logrus](https://github.com/sirupsen/logrus) as the log library please consider using `log.WithError(err).Errorln("Unable to update Core")`.